### PR TITLE
adding datadog and ougai

### PR DESCRIPTION
**Pull request by AlyHKafoury**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,16 +1,19 @@
 # frozen_string_literal: true
 require_relative 'boot'
-
+require 'rails/all'
```
Comment (jbirdjavi) [inline config/application.rb]: Why did you add this line?  This shouldn't be here for this app.

File: config/initializers/datadog.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Datadog.configure do |c|
```
Comment (jbirdjavi) [inline config/initializers/datadog.rb]: This file already exists as datadog_trace.rb

File: config/application.rb (hunk 1 above)
Comment (AlyHKafoury) [reply config/application.rb]: it wouldn't run without it after adding the gems

Comment (AlyHKafoury) [reply config/application.rb]: removed
